ingress/src/main/resources/application.properties

```diff
@@ -15,3 +15,4 @@ mp.messaging.outgoing.events-out.connector=smallrye-kafka
 mp.messaging.outgoing.events-out.topic=events
 mp.messaging.outgoing.events-out.value.serializer=org.apache.kafka.common.serialization.StringSerializer
 
+# event-bridge.bridge.id has to be set by the shard operator in the deployment
```

Comment:
Wdym with this comment here in this file?

Reply:
I mean that I do not provide a default value in the `application.properties`. When the shard creates a Deployment for this ingress application has to provide the `EVENT_BRIDGE_BRIDGE_ID` environment variable to configure it